libraries/api/include/golos/api/comment_api_object.hpp

```diff
@@ -23,9 +23,9 @@ namespace golos { namespace api {
 
         std::string category;
 
-        time_point_sec last_update;
+        fc::optional<time_point_sec> last_update;
         time_point_sec created;
-        time_point_sec active;
+        fc::optional<time_point_sec> active;
```

Comment:
These values now optional, initialization of them needed for case of deleted comment - see, function `discussion_helper::get_discussion(author)`;